src/arithmetic.jl

```diff
@@ -665,6 +665,19 @@ function /(a::Taylor1{TaylorN{T}}, b::Taylor1{TaylorN{T}}) where {T<:NumberNotSe
     return res
 end
 
+function /(a::NumberNotSeries, b::Taylor1{TaylorN{T}}) where {T<:NumberNotSeries}
+    iszero(a) && !iszero(b) && return zero(a)
+
+    # order and coefficient of first factorized term
+    # In this case, since a[k]=0 for k>0, we can simplify to:
+    # ordfact, cdivfact = 0, a/b[0]
+
+    res = Taylor1(a/b[0], b.order)
+    for ordT in eachindex(res)
+        div!(res, a, b, ordT)
+    end
+    return res
+end
```

Comment:
I propose the following here to ensure type stability (this is untested)
```julia
function /(a::NumberNotSeries, b::Taylor1{TaylorN{T}}) where {T<:NumberNotSeries}
    R = promote_type(typeof(a), TaylorN{T})
    iszero(a) && !iszero(b) && return convert(Taylor1{R}, zero(b))

    # order and coefficient of first factorized term
    # In this case, since a[k]=0 for k>0, we can simplify to:
    # ordfact, cdivfact = 0, a/b[0]

    res = Taylor1(a/b[0], b.order)
    for ordT in eachindex(res)
        div!(res, a, b, ordT)
    end
    return res
end
```

Reply:
Note that `a::NumberNotSeries` is perhaps too strong, because it leaves outside `a::TaylorN{T}`, which is a method worth having too.

Reply:
Thanks for the suggestion! I added the fix for type-stability and widened the type for `a` to allow for `TaylorN{T}`